configure/os/CONFIG.freebsd-x86.freebsd-x86

```diff
@@ -4,7 +4,75 @@
 #-------------------------------------------------------
 
 # Include common gnu compiler definitions
-include $(CONFIG)/CONFIG.gnuCommon
+#include $(CONFIG)/CONFIG.gnuCommon
```

Comment:
What necessitates (almost?) duplicating the contents of `CONFIG.gnuCommon` twice?

Reply:
@mdavidsaver :  Thanks for the rewiew. The code duplication was removed now.